src/multimap_table.rs

```diff
@@ -585,15 +585,11 @@ impl<'db, 'txn, K: RedbKey + 'static, V: RedbKey + 'static> MultimapTable<'db, '
     /// Add the given value to the mapping of the key
     ///
     /// Returns `true` if the key-value pair was present
-    pub fn insert<'a>(
+    pub fn insert<'k, 'v>(
         &mut self,
-        key: impl Borrow<K::SelfType<'a>>,
-        value: impl Borrow<V::SelfType<'a>>,
-    ) -> Result<bool>
-    where
-        K: 'a,
-        V: 'a,
-    {
+        key: impl Borrow<K::SelfType<'k>>,
+        value: impl Borrow<V::SelfType<'v>>,
+    ) -> Result<bool> {
```

Comment:
Can you add the `where` clause back? I'm a little unclear on whether it's required for the GAT, but I think it's recommended

Reply:
Adding it back is trivial. But it's redundant because there is `K: RedbKey + 'static, V: RedbKey + 'static` lifetime bound and `K: 'any, V: 'any` always holds.

Reply:
Ah, yes I see. Thanks for pointing that out. Will merge it as is!